### PR TITLE
Added additional project types.

### LINKED ISSUE #20: Update project type guids to be more complete/accurate
The [project type guid constants](https://github.com/VsixCommunity/Community.VisualStudio.Toolkit/blob/master/src/Community.VisualStudio.Toolkit.Shared/Helpers/ProjectTypes.cs) are a great start.  I think some of them might be slightly off on their meaning (for example, the guid associated with the `WAP` constant is really just a `C#` project) and there are a ton more that can be added that I saw [here](https://www.codeproject.com/Reference/720512/List-of-Visual-Studio-Project-Type-GUIDs).

I can open a PR if it's deemed useful to add them.

### PR DESCRIPTION
**Pull request by StevenRasmussen**
Fixes #20 - Please review the names to see if any should be changed.  I basically just took them all from [here](https://www.codeproject.com/Reference/720512/List-of-Visual-Studio-Project-Type-GUIDs) and changed the names.  Many of them I don't even know personally so maybe others have better names/descriptions.  I really only have a question on the `SOLUTION_FOLDER` AND `SOLUTION_FOLDER_OTHER`.  I've detected for the `SOLUTION_FOLDER` before in an extension but have never encountered the `OTHER` one.

### CODE REVIEW

File: src/Community.VisualStudio.Toolkit.Shared/Helpers/ProjectTypes.cs [hunk 1]
```diff
@@ -5,31 +5,208 @@
...
+        public const string DEPLOYMENT_SETUP = "{978C614F-708E-4E1A-B201-565925725DBA}";
+
+        /// <summary>The project type of Deployment Smart Device CAB</summary>
+        public const string DEPLOYMENT_SMART_DEVICE_CAB = "{AB322303-2255-48EF-A496-5904EB18DA55}";
+
+        /// <summary>The project type of Distributed System</summary>
+        public const string DISTRIBUTED_SYSTEM = "{F135691A-BF7E-435D-8960-F99683D2D49C}";
 
         /// <summary>The project type of .NET Core projects.</summary>
         public const string DOTNET_Core = "{9A19103F-16F7-4668-BE54-9A1E7A4F7556}";
```
Comment (madskristensen) [inline src/Community.VisualStudio.Toolkit.Shared/Helpers/ProjectTypes.cs:54]: Casing

**Comment by madskristensen** [comment]
The CI build failed

File: src/Community.VisualStudio.Toolkit.Shared/Helpers/ProjectTypes.cs (hunk 1 above)
Comment (StevenRasmussen) [reply src/Community.VisualStudio.Toolkit.Shared/Helpers/ProjectTypes.cs]: Updated

**Comment by StevenRasmussen** [comment]
Doh!  I didn't realize some of the constants were being used internally.  Updated and should build now ;)

**Comment by madskristensen** [comment]
Awesome. Must have taken you a long time to get that all into the .cs file. Thank you so much!!

**Pull request merged**